test/ItspServices.pServer.Test/AuthorizedEndpointTest.cs

```diff
@@ -171,7 +123,19 @@ public void TestInit()
         [TestMethod]
         public async Task ProtectedDataGetRootFolder()
         {
-            ProtectedDataRepository.Setup(x => x.GetFolderById(null)).Returns(RootFolder);
+            Folder rootFolder = new Folder
+            {
+                Id = 0,
+                Name = "root",
+                Subfolder = new[] {
+                    new Folder
+                    {
+                        Id = 1,
+                        Name = "Folder1"
+                    }
+                }.ToList()
```

Comment:
I would leave out this data since it should never be needed if the user is simply requesting the root directory. This test should verify if the client arrives at the correct endpoint even without specifying an id. And per business logic requirements the response should contain information about the root directory and the root does not necessarily need subfolders.